packages/elements/src/react/common/form/index.tsx

```diff
@@ -528,6 +530,10 @@ type FormInputProps =
  */
 const Input = React.forwardRef<React.ElementRef<typeof RadixControl>, FormInputProps>(
   (props: FormInputProps, forwardedRef) => {
+    const telemetry = useTelemetry();
+
+    telemetry?.record(eventComponentMounted('Elements_Input', props));
+
```

Comment:
❓
1. Wouldn't this fire on every re-render ?
2. Seeing the implementation of `eventComponentMounted` the props you seem to be interested will be ignored.

Reply:
1. Yes. https://github.com/clerk/javascript/pull/3267 is fixing this
2. Thanks! I fixed it by introducing a new helper function

Reply:
They will probably be released around the same time so it's not blocking